### PR TITLE
Config file updates for NUgus vision

### PR DESCRIPTION
**Pull request by Bidski**
Includes some minor code changes and removes commented out debug code.

### CODE REVIEW

File: module/vision/BallDetector/data/config/igus1/BallDetector.yaml [hunk 1]
```diff
@@ -1,19 +1,26 @@
...
-  minimum_points_for_consensus: 10
-  maximum_iterations_per_fitting: 100
-  maximum_fitted_models: 10
-  consensus_error_threshold: 0.0001
-maximum_disagreement_ratio: 1.2
+  minimum_points_for_consensus: 10 
+  maximum_iterations_per_fitting: 10
+  maximum_fitted_models: 3
+  consensus_error_threshold: 0.000001
+visual_mesh:
```
Comment (TrentHouliston) [inline module/vision/BallDetector/data/config/igus1/BallDetector.yaml]: Why is this here?

File: module/vision/BallDetector/data/config/igus1/BallDetector.yaml [hunk 2]
```diff
@@ -1,19 +1,26 @@
...
-green_radial_samples: 12
+green_ratio_threshold: 0.1
+green_radial_samples: 18
 green_angular_samples: 12
+ball_screen_covariance: [0.1, 0.01, 0.01]
 
 print_throwout_logs: false
 
+ball_angular_cov: [0.1,0.01,0.01]
+
```
Comment (TrentHouliston) [inline module/vision/BallDetector/data/config/igus1/BallDetector.yaml]: Why so much whitespace!

File: module/vision/BallDetector/data/config/igus1/BallDetector.yaml [hunk 3]
```diff
@@ -1,19 +1,26 @@
...
+
 clustering:
```
Comment (TrentHouliston) [inline module/vision/BallDetector/data/config/igus1/BallDetector.yaml]: Pretty sure this is no longer used

File: module/vision/LUTClassifier/src/BallEnhancer.cpp [hunk 4]
```diff
@@ -247,23 +248,18 @@ namespace vision {
...
+            if ((segment.segmentClass == ClassifiedImage::SegmentClass::GOAL)
+                && (visualHorizonAtPoint(classifiedImage, segment.start[0]) <= segment.start[1]
+                    || visualHorizonAtPoint(classifiedImage, segment.end[0]) <= segment.end[1])
+                && segment.length > 1) {
+                points.push_back(segment.midpoint);
             }
         }
 
-        // std::vector<std::tuple<Eigen::Vector2i, Eigen::Vector2i, Eigen::Vector4d>,
-        // Eigen::aligned_allocator<std::tuple<Eigen::Vector2i, Eigen::Vector2i, Eigen::Vector4d>>> debug; // DEBUG LINE
```
Comment (TrentHouliston) [inline module/vision/LUTClassifier/src/BallEnhancer.cpp]: This is important! Keep it or make it better

Comment (MattAmos) [reply module/vision/LUTClassifier/src/BallEnhancer.cpp]: I'm pretty sure I just meant to move it onto the same line, so the `//DEBUG LINE` comments actually worked, but clang format got to it

File: module/vision/BallDetector/data/config/igus1/BallDetector.yaml (hunk 3 above)
Comment (MattAmos) [reply module/vision/BallDetector/data/config/igus1/BallDetector.yaml]: It's still being attached to the ball measurement, could be removed in a future pull request

Comment (TrentHouliston) [reply module/vision/BallDetector/data/config/igus1/BallDetector.yaml]: The clustering?

Comment (MattAmos) [reply module/vision/BallDetector/data/config/igus1/BallDetector.yaml]: I misread, thought you meant the ball angular covariance
